### PR TITLE
Dev 2.11

### PR DESCRIPTION
**Pull request by taojing2002**

### CODE REVIEW

File: metacat-index/src/main/java/edu/ucsb/nceas/metacat/index/annotation/EmlAnnotationSubprocessor.java [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+ * BaseXPathDocumentSubprocessor but a bit different in that it post-processes
+ * results from the usual ISolrDoc.getFields() instead of mergingn them asi-s.
+ *
+ * User: Mecum
+ * Date: 2019/02/05
+ * Time: 4:57 PM
+ *
+ */
+public class EmlAnnotationSubprocessor implements IDocumentSubprocessor {
+    private static Logger log = Logger.getLogger(EmlAnnotationSubprocessor.class.getName());
```
[the remark this answers is not in the document]
Comment (amoeba) [reply metacat-index/src/main/java/edu/ucsb/nceas/metacat/index/annotation/EmlAnnotationSubprocessor.java]: Good catch, filed at https://github.com/NCEAS/metacat/issues/1384.

File: metacat-index/src/main/java/edu/ucsb/nceas/metacat/index/annotation/OntologyModelService.java [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+import com.hp.hpl.jena.query.QueryExecution;
+import com.hp.hpl.jena.query.QueryExecutionFactory;
+import com.hp.hpl.jena.query.QueryFactory;
+import com.hp.hpl.jena.query.QuerySolution;
+import com.hp.hpl.jena.query.ResultSet;
+
+import org.dataone.cn.indexer.parser.ISolrDataField;
+import org.dataone.cn.indexer.annotation.SparqlField;
+import org.dataone.configuration.Settings;
+
```
[the remark this answers is not in the document]
Comment (amoeba) [reply metacat-index/src/main/java/edu/ucsb/nceas/metacat/index/annotation/OntologyModelService.java]: Good catch, filed at https://github.com/NCEAS/metacat/issues/1384.

File: build.xml [hunk 3]
```diff
@@ -788,6 +794,15 @@
 				<include name="*.xsd" />
 			</fileset>
 		</copy>
+		
+		<!-- Checkout eml211 for given schema tag-->
```
[the remark this answers is not in the document]
Comment (taojing2002) [reply build.xml]: fixed it.

File: src/edu/ucsb/nceas/metacat/oaipmh/provider/server/catalog/MetacatCatalog.java [hunk 4]
```diff
@@ -516,7 +516,7 @@ private boolean isIncludedDoctype(String doctype, String metadataPrefix) {
      */
     if (doctype != null && 
         (metadataPrefix.equals("oai_dc") ||
-         (doctype.startsWith("eml://ecoinformatics.org/eml-") && 
+         (doctype.contains("//ecoinformatics.org/eml-") &&
```
[the remark this answers is not in the document]
Comment (taojing2002) [reply src/edu/ucsb/nceas/metacat/oaipmh/provider/server/catalog/MetacatCatalog.java]: fixed it.

**Pull request merged**